code/__DEFINES/dcs/signals/signals_global.dm

```diff
@@ -0,0 +1,84 @@
+// Global signals. Format:
```

Comment:
You can split global signals into their own folder too, doesn't have to be in one big file

Reply:
i'd keep the ones with GLOB in the name and the "!..." in here, i'll move the others